src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -265,6 +265,12 @@ void hwm_EvaluationCompleted(object sender, EvaluationCompletedEventArgs e)
         /// </summary>
         internal void UpdateRunStatusMsgBasedOnStates()
         {
+            if ((bool)((Model as HomeWorkspaceModel)?.RunSettings.ForceAutomaticWithoutRun))
```

Comment:
won't this cast fail if Model is a customWorkspaceModel? Or will null cast to false?

Reply:
@mjkkirschner I did confirm that this function is bind to graph evaluation so the above case would not happen